acceptance/terrafarm/farmer.go

```diff
@@ -131,7 +132,7 @@ func (f *Farmer) CollectLogs() {
 		dest += strconv.Itoa(i)
 		if err := f.scp(host, f.defaultKeyFile(), src,
 			filepath.Join(f.AbsLogDir(), dest)); err != nil {
-			fmt.Fprintf(os.Stderr, "error collecting %s from host %s: %s", src, host, err)
+			f.logf("error collecting %s from host %s: %s", src, host, err)
```

Comment:
the log lines here are already pretty long (think output from `apt-get`) and the logger we used is fairly cockroach-specific, so I didn't feel it was necessary here (or rather, this is more flexible than hard-coding the logger - in the desired use case here, all I want is control what goes to `stderr`).